config/config.go

```diff
@@ -690,9 +690,6 @@ func (cfg *MempoolConfig) ValidateBasic() error {
 	if cfg.MaxBatchBytes < 0 {
```

Comment:
IMO MaxBatchBytes must be also greater or equals to MaxTxBytes and since we know that MacBatchBytes higher than N creates a problematic behavior, Tehdernmint should at least print a Warning if MaxBatchBytes is set to a higher value by the user.

Note that with this default configuration, a Tx higher than 4KB would be never shared not received by any node.

Reply:
Yeah, this isn't intuitive to me. The behavior here seems to be that we use the max of _either_ `MaxBatchBytes` or the largest tx in the mempool as the limit. Which means that if there is a tx of e.g. 1 MB in the mempool, then we'll send a 1 MB batch even though the user set the limit at 4 KB.

Reply:
> Note that with this default configuration, a Tx higher than 4KB would be never shared not received by any node.

nice catch! I was trying to come up with a way for batching to still work, but I guess it's better to disable it entirely.